client/src/components/Menu/MenuItem.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Please rename it into `client/src/components/Menu/MenuItem/MenuItem.js`, move css file accordingly and create `package.json` file by example from other components